Dockerfile

```diff
@@ -105,6 +115,12 @@ COPY --from=argo-ui ui/dist/app ui/dist/app
 # stop make from trying to re-build this without yarn installed
 RUN touch ui/dist/node_modules.marker
 RUN touch ui/dist/app/index.html
+
+# Tell git to forget about all of the files that were not included because of .dockerignore in order to ensure that
+# the git state is "clean" even though said .dockerignore files are not present
+RUN cat .dockerignore >> .gitignore
+RUN git status --porcelain | cut -c4- | xargs git update-index --skip-worktree
+
```

Comment:
Because of `.dockerignore` a lot of files were not copied to the builder, leading git to believe they were removed and creating a "dirty" tree state. With this solution we make sure to ignore said files are not present. Note that this is only for said files, if other changes still exist, we will still mark them as diry

Reply:
3x copy and paste - anyway to refactor?